src/Containers/Microsoft.NET.Build.Containers/BuiltImage.cs

```diff
@@ -24,28 +24,43 @@ internal readonly struct BuiltImage
     internal required string ImageSha { get; init; }
 
     /// <summary>
-    /// Gets image size.
+    /// Gets image manifest.
     /// </summary>
-    internal required long ImageSize { get; init; }
+    internal required string Manifest { get; init; } 
 
     /// <summary>
-    /// Gets image manifest.
+    /// Gets manifest digest.
     /// </summary>
-    internal required ManifestV2 Manifest { get; init; }
+    internal required string ManifestDigest { get; init; }
 
     /// <summary>
     /// Gets manifest mediaType.
     /// </summary>
     internal required string ManifestMediaType { get; init; }
 
+    /// <summary>
+    /// Gets image layers.
+    /// </summary>
+    internal List<ManifestLayer>? Layers { get; init; }
+
+    /// <summary>
+    /// Gets image OS.
+    /// </summary>
+    internal string? OS { get; init; }
+
+    /// <summary>
+    /// Gets image architecture.
+    /// </summary>
+    internal string? Architecture { get; init; }
+
     /// <summary>
     /// Gets layers descriptors.
     /// </summary>
     internal IEnumerable<Descriptor> LayerDescriptors
     {
         get
         {
-            List<ManifestLayer> layersNode = Manifest.Layers ?? throw new NotImplementedException("Tried to get layer information but there is no layer node?");
+            List<ManifestLayer> layersNode = Layers ?? throw new NotImplementedException("Tried to get layer information but there is no layer node?");
```

Comment:
I'm not sure if this should be localized

Reply:
same here